(docs): add alert boxes (#7074)
EddieHubCommunity/LinkFree: pages/docs/quickstart.mdx

You can skip `optional extended message`

<Alert
  type="warning"
  message={
    <>

>       <strong>NOTE</strong>: Please use the online{" "}
>       <Link href="https://jsonlint.com">JSON Lint</Link> tool to check if your
>       JSON file is <i>error free</i> before committing your pull request.

Reviewer: ```suggestion
      <strong>NOTE</strong>: Please use our online{" "}
      <Link href="https://linkfree.io/playground">Playground</Link> tool to check your
      JSON file is <I>valid</i> before committing your pull request.
```

>       JSON file is <I>valid</i> before committing your pull request.

Reviewer: ```suggestion
      JSON file is <i>valid</i> before committing your pull request.
```
